fppanalysis/two_dim_velocity_estimates.py

```diff
@@ -1,6 +1,86 @@
 import fppanalysis.time_delay_estimation as tde
 import numpy as np
 import xarray as xr
+from dataclasses import dataclass
+
+
+@dataclass
+class PixelData:
+    """Data class containing estimated data from a single pixel.
+
+    vx: Radial velocities
+    vy: Poloidal velocities
+    confidences:
+        if method='cross_corr':
+            Maximum value of the cross-correlations at each pixel.
+        if method='cond_av':
+            Conditional variance value at maximum cross conditional average for each pixel.
+    R: Radial positions
+    Z: Poloidal positions
+    """
+
+    r_pos: float = 0
+    z_pos: float = 0
+    vx: float = 0
+    vy: float = 0
+    confidence: float = 0
+    events: int = 0
+
+
+class MovieData:
+    """Class containing estimated data for all pixels in a set.
+
+    Use getters to retrieve:
+        vx: Radial velocities
+        vy: Poloidal velocities
+        confidences:
+            if method='cross_corr':
+                Maximum value of the cross-correlations at each pixel.
+            if method='cond_av':
+                Conditional variance value at maximum cross conditional average for each pixel.
+        R: Radial positions
+        Z: Poloidal positions
```

Comment:
What is the format of each variable? Do rows refer to radial positions and columns to the poloidal position? Is it the same format as when we extract ds from cmod_function? Maybe this can be specified

https://github.com/sajidah-ahmed/cmod_functions/blob/main/guides/how_to_use_raw_APD_datatset_function.ipynb

Reply:
This code is in principle agnostic of radial/poloidal directions (even though the name, and the intention is oriented towards that). The indexing should match that of the dataset the user uses to compute the MovieData object. I added a sentence explaining that. Is this enough?

Reply:
Ja det er sant :)